extensions/amp-analytics/0.1/vendors.js

```diff
@@ -24,20 +24,27 @@ export const ANALYTICS_CONFIG = {
   'googleanalytics': {
     'host': 'www.google-analytics.com',
     'method': 'GET',
+    'vars': {
```

Comment:
How does this set reconcile against UrlReplacements? And why does it have some but not other vars from there?